security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort.inc

```diff
@@ -298,7 +276,7 @@ function snort_build_list($snortcfg, $listname = "", $whitelist = false, $extern
 		// List. So for Pass Lists, we test for null strings from
 		// filter_expand_alias() and assume the passed alias is a
 		// FQDN or other dynamically updated alias.
-		if (is_array($list['address']['item']) && count($list['address']['item']) > 0) {
+		if (array_get_path($list, 'address/item')) {
 			foreach ($list['address']['item'] as $addr) {
```

Comment:
Similar to other comments, you can skip the if test by passing the default, and `foreach` should be using `array_get_path()` anyhow, so the end result is:

```php
foreach (array_get_path($list, 'address/item', []) as $addr) {
```